airflow/www/views.py

```diff
@@ -3227,9 +3227,11 @@ class VariableModelView(AirflowModelView):
 
     def hidden_field_formatter(self):
         """Formats hidden fields"""
+        from airflow.utils.log import secrets_masker
```

Comment:
Why can’t this be imported globally?

Reply:
It possibly could, I've just got somewhat "alergic" to global imports in airflow.www. I'll see (I think this one is fine)

Reply:
Done